database/db.js

```diff
@@ -33,18 +38,21 @@ const getReportForUrl = async (url) => {
     return getFirstRow(sql, [url]);
 };
 
+const valReportSummaryOnly = `(SELECT case when val.report is null then null else jsonb_build_object(
```

Comment:
@akmiller01 wondering if this is a legit practice. I know you aren't supposed to use string concatenation to protect from SQL injection, but since this is static I think it should be okay?

Reply:
Readability. It's also used in 3 queries so shortens the overall length.